processing/src/main/java/org/apache/druid/query/aggregation/AggregatorFactory.java

```diff
@@ -94,20 +94,44 @@ public AggregateCombiner makeNullableAggregateCombiner()
   }
 
   /**
-   * Returns an AggregatorFactory that can be used to combine the output of aggregators from this factory.  This
-   * generally amounts to simply creating a new factory that is the same as the current except with its input
-   * column renamed to the same as the output column.
+   * Returns an AggregatorFactory that can be used to combine the output of aggregators from this factory. It is used
+   * when we know we have some values that were produced with this aggregator factory, and want to do some additional
+   * combining of them. This happens, for example, when merging query results from two different segments, or two
+   * different servers.
+   *
+   * The combining factory is often computed by simply creating a new factory that is the same as the current, except
+   * with its input column renamed to the same as the output column. For example, this aggregator:
+   *
+   * {@code {"type": "longSum", "fieldName": "foo", "name": "bar"}}
```

Comment:
I'm not sure `{}` inside `{@code }` work. Anyway, we don't need this, because these docs are not published and are always consumed when reading this code.

Reply:
It does work, I just tested it. But, nevertheless, I removed the code tag and just left the example by itself.